### ISSUE TITLE
Vendor "Currently Equipped" Panel

### ISSUE THREAD

**Issue by Kralomax** [AeroScripts/QuestieDev#300]
When opening a vendors list of items to mouseover, the "currently equipped" item panel would hang/remain open after closing the vendor. Disabling questie removed this problem.
The panel would also randomly attach itself to other frames and remain there until I reloaded. This issue happened with versions 3.1, 3.2, and 3.3. v3.0 worked fine. 
I'm not sure if this was an issue with another addon, but I have listed mine.
https://www.youtube.com/watch?v=WWGC4lIkSms&feature=youtu.be
[Addons.docx](https://github.com/AeroScripts/QuestieDev/files/658882/Addons.docx)

**Comment by AeroScripts** [comment]
Hmm. I have noticed this issue myself but assumed it was related to the broken changes in v3.4

I'm currently in the process of rewriting the tooltip hooks, should be fixed soon.

**Comment by Dzehn** [comment]
I just spend an hour figuring out where this came from as I just installed a new AH add on, Questie was literally the last place I checked.

Btw, it does have something to do with other addons. since if Questie is the only thing active, there are no issues.

for me the issue arrives -Only- When I have "Aux" (AH mod) active.

None of my other addons interfere.

Thank you for already being on the issue. Keep up the amazing work mate.

//Zehn
